src/Popover/Popover.test.js

```diff
@@ -90,6 +106,11 @@ describe('<Popover />', () => {
         component = renderer.create(popOverNoArrow);
         tree = component.toJSON();
         expect(tree).toMatchSnapshot();
+
+        // popover with no disable edge detection
+        component = renderer.create(popOverDisableEdgeDetection);
+        tree = component.toJSON();
+        expect(tree).toMatchSnapshot();
```

Comment:
I'm not sure if this actually tests anything. I can't think of a useful unit test for this, perhaps it's time to bite the bullet and set up puppeteer? Thoughts @greg-a-smith ?

Reply:
I agree that this test doesn't really do anything.  The only way to visually test this prop is to open the popover and scroll to a point where it should flip and then verify it doesn't.

Since that functionality is really core to popper.js, we could probably get away with a test that makes sure our internal Popper component gets this value passed to it and the modifiers are set correctly.  Let me know if you need/want more explanation on that.